### ISSUE TITLE
Unable to replicate Cellpose mAP cited in paper

### ISSUE THREAD

**Issue by NathanMalta** [MouseLand/cellpose#782]
Hi Cellpose team,

I'm having some difficulty replicating the test set mAP recorded in the [Cellpose v1 paper](https://www.nature.com/articles/s41592-020-01018-x).  I'm getting 0.5 mAP of ~0.5 on the cytoplasm dataset while I believe your paper cites around 0.75 (fig 4g).  Here is a snippet of code that I'm using to run the model:

```
model = models.CellposeModel(gpu=use_GPU, model_type='cyto')
channels = [[0,0]]
masks, flows, styles = model.eval(imgs_2D, diameter=None, flow_threshold=None, channels=channels)
```

As an aside, I am converting all images in the dataset to grayscale to be comparable to other methods, and I add a border such that all images are the same size.  While this might have a slight effect, I wouldn't expect it to be so drastic, but let me know if you disagree.

Thanks

**Comment by carsen-stringer** [comment]
We used the test images at their original sizes, and estimated the diameters. The `Cellpose` model runs both the size estimation and the mask estimation, so the current way to run the benchmark would be:
```
from cellpose import models, io, metrics

img_test = [io.imread(root / "test" / f"{i:03d}_img.tif")
                 for i in range(68)]
masks_test = [io.imread(root / "test" / f"{i:03d}_masks.tif")
                 for i in range(68)]
seg_model = models.Cellpose(gpu=True, model_type="cyto", net_avg=True)
masks = seg_model.eval(img_test, diameter=None, channels=[1,2], channel_axis=0, normalize=True)[0]
ap, tp, fp, fn = metrics.average_precision(masks_test, masks)
print(ap.mean(axis=0))

```

Note we always provide two channels, we do not average the nuclear and cytoplasm channel (that's what [0,0] does) -- that results in worse performance. Also the `flow_threshold` should be set to `0.4` (the default value). And estimating the diameter is crucial (the way you ran it the diameter is set to 30 pixels for all images).
